src/custom/pages/Claim/ClaimSummary.tsx

```diff
@@ -19,16 +20,25 @@ export function ClaimSummary({ hasClaims, isClaimed, unclaimedAmount }: ClaimSum
   const { chainId } = useActiveWeb3React()
   const { activeClaimAccount, claimStatus, isInvestFlowActive } = useClaimState()
 
-  const vCowBalance = useTokenBalance(activeClaimAccount || undefined, chainId ? V_COW[chainId] : undefined)
+  const vCow = chainId ? V_COW[chainId] : undefined
+
+  const vCowBalance = useTokenBalance(activeClaimAccount || undefined, vCow)
 
   const hasClaimSummary = claimStatus === ClaimStatus.DEFAULT && !isInvestFlowActive
 
-  if (!hasClaimSummary) return null
+  if (!hasClaimSummary || !vCow) return null
+
+  let totalAvailableAmount: CurrencyAmount<Token> | undefined = CurrencyAmount.fromRawAmount(vCow, JSBI.BigInt(0))
```

Comment:
This is now set to CurrencyAmount object by default because we want to show zero in this case and so that `formatSmartLocaleAware` and `formatMax` work